doc/classes/String.xml

```diff
@@ -184,6 +184,22 @@
 				Returns a string with [param chars] characters erased starting from [param position]. If [param chars] goes beyond the string's length given the specified [param position], fewer characters will be erased from the returned string. Returns an empty string if either [param position] or [param chars] is negative. Returns the original string unmodified if [param chars] is [code]0[/code].
 			</description>
 		</method>
+		<method name="filecasecmp_to" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="to" type="String" />
+			<description>
+				Like [method naturalcasecmp_to] but sorts strings starting in [code]_[/code] before those starting in characters other than [code].[/code].
```

Comment:
I would typically be against saying anything along the lines of "Look at this method to understand how this ones works". But in this case there's a lot to say and the difference is minimal, so okay!
"Begins with" is consistent with the method called the same way.
```suggestion
				Like [method naturalcasecmp_to] but prioritises strings that begin with periods ([code].[/code]) and underscores ([code]_[/code]) before any other character. Useful when sorting folders or file names.
```
(Similar suggestion for `filenocasecmp_to` below. Applies to StringName as well)

Reply:
The period part is already in `natural` comparisons, but applying this, thank you!